packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -645,7 +645,7 @@ export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implem
       body: JSON.stringify({
         corpus_id: this.corpusId,
         query,
-        page_size: params?.limit ?? 10,
+        max_chunks: params?.limit ?? 10,
```

Comment:
We should really be using camelCase I think, but if this works then I suppose I don't really care either way. :)

Reply:
i don't know what the right way is. i just copied the format that was used here https://github.com/fixie-ai/fixie/blob/4d7f23adaa585a01e408f8f68a05574800a9db08/src/proto_api/fixie_api/corpus/v1/service.proto#L152C22-L152C22

Reply:
Fair enough. The OpenAPI spec we generate uses camelCase for field names, but our server will parse either one.